engine/gamesys/src/gamesys/components/comp_private.cpp

```diff
@@ -350,11 +352,6 @@ void SetRenderConstant(HComponentRenderConstants constants, dmRender::HMaterial
             return;
         }
 
-        if (constants->m_RenderConstants.Full())
-        {
-            constants->m_RenderConstants.OffsetCapacity(1);
-            constants->m_PrevRenderConstants.OffsetCapacity(1);
-        }
         constants->m_RenderConstants.SetSize(count+1);
```

Comment:
A reallocation might make the buffer point to somewhere else, thus leaving any existing pointers into this area, as new dangling pointers.